src/lib/ZoomControls.js

```diff
@@ -0,0 +1,134 @@
+import isFinite from 'lodash/isFinite';
+import EventEmitter from 'events';
+import { ICON_ZOOM_IN, ICON_ZOOM_OUT } from './icons/icons';
+
+const CLASS_ZOOM_CURRENT_SCALE = 'bp-zoom-current-scale';
```

Comment:
minor: Seems like there could be a method that is like `getClasses` that would generate the proper classes for the different type of viewer (if those are required to be different per viewer)

Reply:
I tried to keep this as a reusable component that is agnostic of viewers, so in that sense, I allow the viewers to specify what classnames they want to use